lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -369,20 +369,22 @@ sub create_ecoscore_panel($$$) {
 
         foreach my $adjustment ("production_system", "origins_of_ingredients", "threatened_species", "packaging") {
 
+            my $plus_or_minus;
```

Comment:
difficult to follow, what this is from reading the code, could you add comments ?

Reply:
@jasmeet0817 : right, I changed a few things in the API and forgot to point it out.

I separated the "plus" and "minus" values I initially planned to put in the "grade" field. But then we would have essentially one field for two different thing: an actual grade "a" to "e", and then "plus" or "minus".

So I made a new field, which I called "plus_or_minus" because I failed to find a better name. :) 

I will in fact remove this code from KnowledgePanel.pm and in fact set the value directly in the JSON template for each panel.